baybe/telemetry.py

```diff
@@ -114,7 +114,7 @@
 # Telemetry labels for metrics
 TELEM_LABELS = {
     "RECOMMENDED_MEASUREMENTS_PERCENTAGE": "value_recommended-measurements-percentage",
-    "BATCH_QUANTITY": "value_batch-quantity",
+    "BATCH_SIZE": "value_batch-quantity",
```

Comment:
@Scienfitz: This is the only part where I was not really sure what to do (i.e. see value, not key). Shall we rename that as well or rather keep as is?

Reply:
hmm if we really change it then I guess theres no way around it to change this too, can merge these values in the analysis of telemetry data anyway

Reply:
Alright then, changed as well